cmd/root.go

```diff
@@ -263,6 +265,21 @@ default values.`,
 		"",
 		"Data-dir configures the location of logs and data for validation. This will override the data_directory from configuration file",
 	)
+
+	checkDataCmd.Flags().Int64Var(
+		&tableSize,
+		"table-size",
+		0,
+		`Table-size configures the TableSize for badger DB. If table-size != 0, this will override the table_size from configuration file`,
+	)
+
+	checkDataCmd.Flags().BoolVar(
+		&turnOnMemoryMode,
+		"turn-on-memory-mode",
```

Comment:
this is too long. Consider `in-memory-mode`

Reply:
updated